### ISSUE TITLE
Unix installation instructions missing call to ./bootstrap

### ISSUE THREAD

**Issue by jkbzh** [Cacti/spine#318]
Although the call to bootstrap is mentioned in the cygwin instructions, it is missing in the Unix installation. It should be called before ./configure

https://github.com/Cacti/spine/blob/09e5a8c3c2ffe51a92b03d9f3725c3149ad2abf9/README.md?plain=1#L23

**Comment by TheWitness** [comment]
This one is waiting on this one to be merged into develop.  It's already merged for 1.2.x.

https://github.com/Cacti/spine/pull/319
